.github/workflows/integration.yml

```diff
@@ -0,0 +1,55 @@
+name: Integration tests
+
+on: [push, pull_request]
+
+jobs:
+  Integration:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push' || github.repository == 'DIRACGrid/DIRAC'
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: False
+      matrix:
+        MYSQL_VER:
+          - 5.7
+          - 8.0
+        ES_VER:
+          - 6.6.0
+          - 7.3.2
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Clean up GitHub actions environment
+      run: |
+        set -x
+        mysqladmin -u root -proot shutdown
+
+        # Prepare wrapper script
+        echo "#!/usr/bin/env bash" >> run_in_container
+        echo "set -euo pipefail" >> run_in_container
+        echo "IFS=\$'\n\t'" >> run_in_container
+        echo -n "exec docker exec -w /repo " >> run_in_container
+        # Set environment variables
+        echo -n "-e CI_PROJECT_DIR=/repo " >> run_in_container
+        echo -n "-e CI_COMMIT_REF_NAME=$GITHUB_REF " >> run_in_container
+        echo -n "-e CI_REGISTRY_IMAGE=gitlab-registry.cern.ch/cburr/dirac " >> run_in_container
+        echo -n "-e MYSQL_VER=${{ matrix.MYSQL_VER }} " >> run_in_container
+        echo -n "-e ES_VER=${{ matrix.ES_VER }} " >> run_in_container
+        # Finish wrapper script
+        echo -n "dirac-testing-host \"\$@\"" >> run_in_container
+        chmod +x run_in_container
+    - name: Prepare environment
+      run: |
+        docker run --name dirac-testing-host -v $PWD:/repo --detach --privileged -v /var/run/docker.sock:/var/run/docker.sock --rm gitlab-registry.cern.ch/cburr/dirac/docker-compose:latest bash -c 'sleep 100000000'
```

Comment:
The docker images in `gitlab-registry.cern.ch/cburr/dirac/` are set to regenerate every Sunday based on what is in integration, which is being mirrored from here. The best place for them is probably the [GitHub package registry](https://github.com/features/package-registry), but that's not public yet.